.golangci.yaml

```diff
@@ -69,7 +59,6 @@ linters-settings:
       - name: error-strings
       - name: error-naming
       - name: exported
-        disable-stuttering-check: true
```

Comment:
I'd prefer to keep this rule, it's common best practice for Go style. See https://google.github.io/styleguide/go/decisions#repetition

Reply:
It's included in enable-all :)

Reply:
Revive is not enabled (see L141), and having a stuttering check would be good, right? So this should be fine.